src/order/disjointed.lean

```diff
@@ -1,67 +1,22 @@
 /-
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl
+Authors: Johannes Hölzl, Yaël Dillies
```

Comment:
You could argue you haven't earned authorship yet in this PR :)

Reply:
Oh whoops!